dipy/reconst/ivim.py

```diff
@@ -0,0 +1,391 @@
+""" Classes and functions for fitting ivim model """
+from __future__ import division, print_function, absolute_import
+import numpy as np
+
+from dipy.core.gradients import gradient_table
+from dipy.reconst.base import ReconstModel
+from dipy.reconst.dti import apparent_diffusion_coef
+from dipy.reconst.dti import TensorModel, mean_diffusivity
+from dipy.reconst.multi_voxel import multi_voxel_fit
+from dipy.reconst.vec_val_sum import vec_val_vect
+from dipy.core.sphere import Sphere
+
+from distutils.version import LooseVersion
+import scipy
+
+SCIPY_LESS_0_17 = LooseVersion(scipy.version.short_version) < '0.17'
+
+if not SCIPY_LESS_0_17:
+    least_squares = scipy.optimize.least_squares
+else:
+    leastsq = scipy.optimize.leastsq
+
+
+def ivim_function(params, bvals):
```

Comment:
Can you change the name of this function to ivim_prediction? This function should also have the following inputs: params, gtab and S0. This to keep consistency with the other fitting modules (e.g. see tensor_prediction function of dti.py, or the dki_prediction function of dki.py).